dashboard/src/stores/ingest.ts

```diff
@@ -21,21 +21,21 @@ export interface Pager {
   pages: Array<number>;
 }
 
-export const usePackageStore = defineStore("package", {
+export const useIngestStore = defineStore("ingest", {
   state: () => ({
-    // Package currently displayed.
-    current: null as api.EnduroIngestSip | null,
+    // SIP currently displayed.
+    currentSip: null as api.EnduroIngestSip | null,
```

Comment:
Are you using mixed case "Sip" here because the API uses mixed case?  I'd prefer `currentSIP` like we would do in Go.